core/src/main/scala/org/bitcoins/core/util/Bech32.scala

```diff
@@ -143,17 +143,10 @@ sealed abstract class Bech32 {
           } else {
             val byte = Bech32.charset.indexOf(h.toLower).toByte
 
-            if (byte >= 0 && byte < 32) {
```

Comment:
why did this get removed? Do we have a guarantee that the byte is in the right range?

Reply:
Yes: `Bech32.charset.contains(h.toLower)` is above and provides this garuntee

Reply:
Which is why this was never being covered

Reply:
If `h.toLower` is not in the charset, it will return `-1`. So then this check would fail but now there would be no check

Reply:
no, if `h.toLower` is not in the charset, then the check above this one will throw an `IllegalArgumentException`